src/realm/object-store/sync/sync_manager.hpp

```diff
@@ -227,13 +227,14 @@ class SyncManager : public std::enable_shared_from_this<SyncManager> {
     // Immediately closes any open sync sessions for this sync manager
     void close_all_sessions() REQUIRES(!m_mutex, !m_session_mutex);
 
+    // Set the sync route - it is not allowed to be cleared once set
```

Comment:
should we assert the route is not empty? And also, does this comment mean you cannot set it again? (we should assert on that too then)